### PR TITLE
Update Kubernetes example for v1beta3 API.

### PR DESCRIPTION
**Pull request by enisoc**
@thompsonja Some of this may need some adjustments for the benchmarking versions.

### CODE REVIEW

File: doc/GettingStartedKubernetes.md [hunk 1]
```diff
@@ -467,38 +552,44 @@ provides more detail about how the app server interacts with Vitess.
...
 ## Tear down and clean up
 
-The following command tears down the Container Engine cluster. It is
-necessary to stop the virtual machines running on the Cloud platform.
+Before stopping the Container Engine cluster, you should tear down the Vitess
+services. Kubernetes will then take care of cleaning up any entities it created
+for those services, like external load balancers.
+
+``` sh
+vitess/examples/kubernetes $ ./guestbook-down.sh
```
Comment (thompsonja) [inline doc/GettingStartedKubernetes.md]: Typically tearing down the cluster also tears down all the other services, is this not true in v1beta3 or are we just being more explicit here?

File: doc/GettingStartedKubernetes.md [hunk 2]
```diff
@@ -467,38 +552,44 @@ provides more detail about how the app server interacts with Vitess.
...
+It's also a good idea to remove the firewall rules you created, unless you plan
+to use them again soon:
 
 ``` sh
-$ gcloud compute forwarding-rules delete k8s-example-default-vtctld
-$ gcloud compute forwarding-rules delete k8s-example-default-guestbook
-$ gcloud compute firewall-rules delete vtctld
-$ gcloud compute firewall-rules delete guestbook
-$ gcloud compute target-pools delete k8s-example-default-vtctld
-$ gcloud compute target-pools delete k8s-example-default-guestbook
```
Comment (thompsonja) [inline doc/GettingStartedKubernetes.md]: Does deleting the firewall-rule now delete the forwarding-rule / target pool?

**Comment by thompsonja** [comment]
LGTM

File: doc/GettingStartedKubernetes.md (hunk 1 above)
Comment (enisoc) [inline doc/GettingStartedKubernetes.md]: If you delete the cluster without first removing the services, Kubernetes leaves behind the randomly-named forwarding-rules and target-pools that it created. Those are annoying to clean up now that they're randomly-named, so I figure it's easier to just tell Kubernetes to do it.

File: doc/GettingStartedKubernetes.md (hunk 2 above)
Comment (enisoc) [inline doc/GettingStartedKubernetes.md]: See above.

**Pull request merged**